PatTools/python/patFinalStateProducers.py

```diff
@@ -62,7 +62,7 @@ def produce_final_states(process, daughter_collections, output_commands,
         if 'extraWeights' in src:
             process.patFinalStateEventProducer.extraWeights = src['extraWeights']
         process.patFinalStateEventProducer.trgSrc = cms.InputTag("selectedPatTrigger")
-        process.patFinalStateEventProducer.rhoSrc = cms.InputTag('fixedGridRhoAll')
+        process.patFinalStateEventProducer.rhoSrc = cms.InputTag('fixedGridRhoFastjetAll')
```

Comment:
Do you have a reference to this recommendation? The selection dates from me asking salvatore back in phys14, do we really want Fastjet now?